README.md

```diff
@@ -63,7 +63,57 @@ Follow the official [Firebase guide](https://firebase.google.com/docs/flutter/se
 - Add the `GoogleService-Info.plist` file for iOS and MacOS.
 - Create the `firebase_options.dart` file in the lib folder.
 
+## 📜 Populating Monument Data
 
+This script allows you to populate your Firestore database with predefined monument data. It’s a standalone tool that you can run once to seed your database—no need to run or modify any Flutter app.
+
+### Prerequisites
+
+1. **Node.js**:
+   Install Node.js from [https://nodejs.org](https://nodejs.org).
+   Verify installation: node -v
+   You should see a version number like i.e:
+
+2. **Firebase Project with Firestore Enabled**:
+- Go to [Firebase Console](https://console.firebase.google.com).
+- Create or select a project.
+- Enable Firestore.
+
+3. **Service Account Key**:
+- In the Firebase console, go to "Project Settings" → "Service accounts".
+- Click "Generate new private key" to download `serviceAccountKey.json`.
+- Save `serviceAccountKey.json` in a folder dedicated to running the script (not necessarily inside the Monumento project folder).
+
+### Steps to Populate Data
+
+1. **Set Up a Separate Folder**:
+- Create a new folder (for example `firestore_scripts`) outside of the app directory, such as on your desktop, to organize and manage Firestore-related   scripts independently.
+
+2. **Add the Script and Key**:
+- Place your `serviceAccountKey.json` here.
+- First, create a file named populate_monuments.js in the firestore_scripts folder you created outside of the app directory (e.g., on your desktop).
```

Comment:
I think these steps are not needed anymore, instead we should tell the contributors to simply place the json file in the scripts folder and then run the command from that directory